<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTrackerStabilizerFilteringWidget</class>
 <widget class="QWidget" name="qSlicerTrackerStabilizerFilteringWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="InputTransformWidget">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLLinearTransformNode</string>
        </stringlist>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="removeEnabled">
        <bool>false</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="OutputTransformWidget">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLLinearTransformNode</string>
        </stringlist>
       </property>
       <property name="baseName">
        <string>TrackerStabilizerFiltering</string>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="FilteringBox">
     <property name="title">
      <string>Filter</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Smoother</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Faster</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="ctkDoubleSpinBox" name="DoubleSpinBox">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>50.000000000000000</double>
          </property>
          <property name="value">
           <double>7.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="ctkDoubleSlider" name="FilteringValueWidget">
          <property name="value">
           <double>7.500000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>50.000000000000000</double>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSlider.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>FilteringValueWidget</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DoubleSpinBox</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FilteringValueWidget</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
